parse_lib.py

```diff
@@ -106,6 +106,12 @@ def update_anchor_href(anchor):
         anchor['target'] = '_blank'
 
 
+def convert_to_img(html, equation):
```

Comment:
How about `convert_svg_object_to_img`?  I think there can be objects that are not SVGs.

Also note comments about side-effects.